chartpress.py

```diff
@@ -289,6 +299,8 @@ def main():
         help='extra message to add to the commit message when publishing charts')
     argparser.add_argument('--image-prefix', default=None,
         help='override image prefix with this value')
+    argparser.add_argument('--sanitize', action='store_true',
```

Comment:
Sanitize is a surprising name, and it isn't clear what "clean up image tags" means. Is this resetting to the original state (i.e. the same as `git checkout -- chart`)? If so, I might call it `--reset`, and give a more detailed explanation of what it does.

Reply:
yes, `--reset` may be a better name. It's like a `git checkout -- chart` except that in that case it's annoying to have to juggle commits. 

The idea here is really just to undo the image tag and chart version changes of `chartpress` to keep the diff cleaner. I've found it reasonable to keep image tags at `latest` in the codebase, and only change them in published charts. However, that might not work for everyone so maybe specifying a different "reset" tag/version would be the way to go. However, in that case maybe all that is needed is to have a `--skip-build` option with `--tag`.

Reply:
putting the resetTag in chartpress.yaml would be a way to allow charts to pick what they use (we use an explicitly invalid `set-by-chartpress`, since installing from git without rendering doesn't make sense for our charts)

Reply:
+1 on the `set-by-chartpress`. I will implement the `resetTag` option and rename `--sanitize` to `--reset`.